db/migrate/20150312215548_create_videos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVideos < ActiveRecord::Migration
+  def change
+    create_table :videos do |t|
+      t.string :video_id
+      t.string :youtube_link
+      t.string :title
+      t.integer :priority
```

Comment:
Not sure if this is the priority metric they want. Let's remove this for now and add it in when we've finalized how they want to set priorities.